src/main/java/com/onarandombox/MultiverseCore/commands/GameruleCommand.java

```diff
@@ -0,0 +1,87 @@
+package com.onarandombox.MultiverseCore.commands;
+
+import java.util.ArrayList;
+
+import co.aikar.commands.CommandIssuer;
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.CommandPermission;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Optional;
+import co.aikar.commands.annotation.Subcommand;
+import co.aikar.commands.annotation.Syntax;
+import com.onarandombox.MultiverseCore.MultiverseCore;
+import com.onarandombox.MultiverseCore.api.MVWorld;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.GameRule;
+import org.bukkit.World;
+import org.jetbrains.annotations.NotNull;
+
+@CommandAlias("mv")
+public class GameruleCommand extends MultiverseCoreCommand {
+    public GameruleCommand(@NotNull MultiverseCore plugin) {
+        super(plugin);
+    }
+
+    @Subcommand("gamerule")
+    @CommandPermission("multiverse.core.gamerule")
+    @CommandCompletion("@gamerules True|False|@range:1-10 @mvworlds|*")
+    @Syntax("<Gamerule> <Gamerule value> [World or *]")
+    @Description("Changes a gamerule in 1 world")
```

Comment:
Since you added support for `8*`, ig its one or more worlds?

Reply:
`*` is for all worlds, to fix https://github.com/Multiverse/Multiverse-Core/issues/2814

Reply:
I'm talking about the description

Reply:
Ah, I'll change that!